### PR TITLE
Fixes #35448 - Update roadie-rails to 3.x

### PR DESCRIPTION
**Pull request by ekohl**
This has formal Ruby 3 support, even though there are no code differences. It's a major version because it drops Ruby < 2.6.

### CODE REVIEW

File: Gemfile
Comment (ekohl) [inline Gemfile:38]: https://github.com/theforeman/foreman-packaging/pull/8423

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**,

[v2.3.0...v3.0.0 changes](https://github.com/Mange/roadie-rails/compare/v2.3.0...v3.0.0) looks safe, tests are green, so let's get this in.

Thanks @ekohl !

**Comment by stejskalleos** [comment]
Waiting for approval from @theforeman/packaging

**Pull request merged**
